src/cli/generator/config.rs

```diff
@@ -81,10 +81,13 @@ pub enum Source<Status = UnResolved> {
         is_mutation: Option<bool>,
         field_name: String,
     },
+    #[serde(rename_all = "camelCase")]
     Proto {
         src: Location<Status>,
         url: String,
         #[serde(skip_serializing_if = "Option::is_none")]
+        proto_paths: Option<Vec<Location<Status>>>,
```

Comment:
Based on your use case would it make sense to put `proto_paths` at root level in the `Config` or should it be set for each `Source::Proto`?

Reply:
Because `inputs` can configure multiple proto entry files, I don't think it should be applied to all `Source::Proto`. We can maintain this flexibility and let users configure it themselves. I think this configuration is a bit like `connectRPC`, So I think this is reasonable.